main.go

```diff
@@ -91,13 +94,21 @@ func main() {
 
 	// Create a new manager to manage the operator
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     args.metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: args.metricsAddr,
+		},
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
+		}),
 		HealthProbeBindAddress: args.probeAddr,
 		LeaderElection:         args.enableLeaderElection,
 		LeaderElectionID:       "39f5e5c3.nodefeaturediscoveries.nfd.kubernetes.io",
-		Namespace:              watchNamespace,
+		Cache: cache.Options{
+			DefaultNamespaces: map[string]cache.Config{
+				watchNamespace: cache.Config{},
```

Comment:
We have a `utils.GetWatchNamespace()` is this no longer needed?

Reply:
it is needed. The Namespace parameter of the manager Options in 0.14.0 has been moved into the cache.Options. DefaultNamespaces is a map that can configure all the namespaces to watch  (for cache). We are using the output value of utils.GetWatchNamespace()  as a key in the DefaultNamespaces map